File: app/src/main/java/com/rolandvitezhu/todocloud/network/api/user/dto/ModifyPasswordRequest.java
package com.rolandvitezhu.todocloud.network.api.user.dto;

import com.google.gson.annotations.SerializedName;

public class ModifyPasswordRequest {

  @SerializedName("current_password")
  public String currentPassword;
  @SerializedName("new_password")
  public String newPassword;

  public String getCurrentPassword() {
    return currentPassword;
  }

  public void setCurrentPassword(String currentPassword) {
    this.currentPassword = currentPassword;
  }

  public String getNewPassword() {
    return newPassword;
  }

  public void setNewPassword(String newPassword) {
    this.newPassword = newPassword;
  }
}

File: app/src/main/java/com/rolandvitezhu/todocloud/network/api/user/dto/ModifyPasswordRequest.kt
package com.rolandvitezhu.todocloud.network.api.user.dto

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class ModifyPasswordRequest (
        @SerializedName("current_password")
        var currentPassword: String?,
        @SerializedName("new_password")
        var newPassword: String?
) : Parcelable {
    constructor() : this(
            null,
            null
    )
}